st2client/st2client/commands/action_alias.py

```diff
@@ -0,0 +1,65 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from st2client.models.action_alias import ActionAlias
+from st2client.commands import resource
+from st2client.commands.resource import add_auth_token_to_kwargs_from_cli
+from st2client.formatters import table
+
+__all__ = [
+    'ActionAliasBranch'
+]
+
+
+class ActionAliasBranch(resource.ResourceBranch):
+    def __init__(self, description, app, subparsers, parent_parser=None):
+        super(ActionAliasBranch, self).__init__(
+            ActionAlias, description, app, subparsers,
+            parent_parser=parent_parser, read_only=True,
+            commands={
+                'list': ActionAliasListCommand,
+            })
+
+
+class ActionAliasListCommand(resource.ResourceCommand):
+    display_attributes = ['ref', 'pack', 'name', 'description', 'enabled', 'formats']
```

Comment:
We display arrays as string joined with "," and I think it looks OK, but I'm also fine with dropping it in list.

```bash
python st2client/st2client/shell.py action-alias list
+---------------------------+------+-----------------------+-----------------------------------+---------+-----------------------------------+
| ref                       | pack | name                  | description                       | enabled | formats                           |
+---------------------------+------+-----------------------+-----------------------------------+---------+-----------------------------------+
| st2.st2_actions_list      | st2  | st2_actions_list      | List available StackStorm         |         | st2 list actions, st2 list        |
|                           |      |                       | actions.                          |         | actions from {{ pack }}           |
| st2.st2_executions_get    | st2  | st2_executions_get    | Retrieve details for a single     |         | st2 get execution {{ id }}, st2   |
|                           |      |                       | execution.                        |         | show execution {{ id }}, st2      |
|                           |      |                       |                                   |         | executions get {{ id }}, st2      |
|                           |      |                       |                                   |         | executions show {{ id }}          |
| st2.st2_executions_list   | st2  | st2_executions_list   | List available StackStorm         |         | st2 list executions, st2 list     |
|                           |      |                       | executions.                       |         | executions with status {{ status  |
|                           |      |                       |                                   |         | }}, st2 list executions for       |
|                           |      |                       |                                   |         | action {{ action }}               |
| st2.st2_executions_re_run | st2  | st2_executions_re_run | Re-run an action execution.       |         | st2 re-run execution {{ id }},    |
|                           |      |                       |                                   |         | st2 executions re-run {{ id }}    |
| st2.st2_rules_list        | st2  | st2_rules_list        | List available StackStorm rules.  |         | st2 list rules, st2 list rules    |
|                           |      |                       |                                   |         | from {{ pack }}                   |
| st2.st2_sensors_list      | st2  | st2_sensors_list      | List available StackStorm         |         | st2 list sensors, st2 list        |
|                           |      |                       | sensors.                          |         | sensors from {{ pack }}           |
+---------------------------+------+-----------------------+-----------------------------------+---------+-----------------------------------+
```